### ISSUE TITLE
Figure out what to do with unlabeled data

### ISSUE THREAD

**Issue by thomashopkins32** [thomashopkins32/HuBMAP#7]
There are ~6,000 images in the training set that have no labels. Examine a few a try to determine the best method for utilizing this data.

Look into the following and report back:
- self-supervised learning
- unsupervised pretraining

Also look into how we might use this information in other ways.

**Comment by thomashopkins32** [comment]
I want to try self-supervised learning where we randomly mask out a portion of the image and then predict the missing region. I need to do more research on this topic but I think this might be the easiest way to incorporate the 7,000 unlabeled images.
